src/Raven.Server/Documents/Queries/AST/PatternMatchElementExpression.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Raven.Server.Documents.Queries.AST
+{
+    public class PatternMatchElementExpression : PatternMatchExpression
+    {
+        public enum Direction
```

Comment:
Why do we need direction, isn't that what the `From` / `To` for?

Reply:
We don't, this is mostly for debugging -> I use it in query's ToString()

Reply:
Let's remove that, we don't need this.

`(u1)-[:Rated]->(m)<-[:Rated]-(u2)` is the same as:  `(u1)-[:Rated]->(m) and (u2)-[:Rated]->(m)`